### ISSUE TITLE
[A11y] Dashboard - Keyboard-only access to widgets

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1851]
### Operating System

Windows

### Browser

Edge

### Affected page

Backend

### Admin user?

Yes

### Issue Description

You can't use the keyboard to interact with the various widgets (e.g., "USER ACTIVITY") at the bottom of the page.

- Blind screen reader users can likely get enough data out of the hidden tables, so I think they're okay.
- Sighted keyboard-only users cannot access the charts directly, and won't have any way to see the hidden tables.
- Users who are both sighted and use a screen reader will find the tables confusing since the information in the graphs is presented differently than tabular data.

### Suggested Fix

Something needs to be added to allow keyboard users the same functionality mouse users get.  This is a usability issue as well as an accessibility issue since mobile users usually don't have a keyboard attached, and tapping the tiny targets to see the detailed breakdowns is very difficult on a small device.

Maybe we need a button to show the tables in a modal or something?

**Comment by jechols** [comment]
QA depends on remediation.  If a plain table can be shown, anybody can QA this and verify that the table gives the same data as the graph.

**Comment by azle** [comment]
Updated 1/6/2024

"User Activity" is not an option on my screen however "Repository Growth" and "Visibility, Work Types and Resource Types" are not keyboard navigable.

<img width="1026" alt="Screenshot 2025-01-06 at 1 51 25 PM" src="https://github.com/user-attachments/assets/d6a6aece-8438-4e77-8eec-b76ff0ba9d32" />
